CalendarFXView/src/main/java/com/calendarfx/view/DateControl.java

```diff
@@ -1038,6 +1037,67 @@ public String toString() {
         }
     }
 
+    /**
+     * Possible edit operations with an entry.
+     * This enum will be used as parameter of the callback set with {@link DateControl#setEntryEditPolicy}.
+     */
+    public enum EditOperation {
+
+        /**
+         * Checked if any drag/drop operation is allowed
+         */
+        DRAG_AND_DROP,
```

Comment:
I do not like the use of the term "DRAG_AND_DROP" in the context of the entry edit policy. This is a rather technical concept provided by the platform for moving things from one place to another. However, we are not using the javafx drag and drop infrastructure, we are using standard mouse listeners. This term / enum might confuse the users of the API.

Reply:
Ok. I removed that.